common/PublicNetworks.js

```diff
@@ -3,28 +3,28 @@ module.exports = {
     name: "mainnet",
     ethFaucet: null,
     etherscan: "https://etherscan.io/",
-    daiAddress: "0x89d24A6b4CcB1B6fAA2625fE562bDD9a23260359",
-    wethAddress: "0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2"
+    daiAddress: "0x6B175474E89094C44Da98b954EedeAC495271d0F",
```

Comment:
This changed from [Sai](https://etherscan.io/address/0x89d24A6b4CcB1B6fAA2625fE562bDD9a23260359) -> [Dai](https://etherscan.io/address/0x6B175474E89094C44Da98b954EedeAC495271d0F)